tests/tpot_tests.py

```diff
@@ -357,11 +392,20 @@ def test_invalid_pipeline():
     )
     tpot_obj._optimized_pipeline = creator.Individual.from_string(pipeline_string, tpot_obj._pset)
     tpot_obj.fitted_pipeline_ = tpot_obj._toolbox.compile(expr=tpot_obj._optimized_pipeline)
-    # test _wrapped_cross_val_score with cv=20 so that it is impossible to finish in 1 second
```

Comment:
This comment was inaccurate so I removed it.

Reply:
Thank you for cleaning it.